yazi-fm/src/executor.rs

```diff
@@ -282,8 +282,8 @@ impl<'a> Executor<'a> {
 				"insert" => self.cx.input.insert(exec.named.contains_key("append")),
 				"visual" => self.cx.input.visual(),
 
-				"backward" => self.cx.input.backward(),
-				"forward" => self.cx.input.forward(exec.named.contains_key("end-of-word")),
+				"backward" => self.cx.input.move_word(false, false),
```

Comment:
We need to keep the function name matches the command name, which allows us to implement some validation through macros like:

https://github.com/sxyazi/yazi/blob/521c8b6bfefbaa70b8564812ee377f219871bbe1/yazi-fm/src/executor.rs#L223-L234